tests/e2e_tests/conftest.py

```diff
@@ -67,10 +68,11 @@ def authenticator_fund_config(request: FixtureRequest) -> TestFundConfig:
 
 
 @pytest.fixture
-def context(request: FixtureRequest, browser: Browser, e2e_test_secrets: EndToEndTestSecrets):
+def context(new_context: CreateContextCallback, request: FixtureRequest, e2e_test_secrets: EndToEndTestSecrets):
```

Comment:
question: is it just a syntax change here? Injecting directly `CreateContextCallback` so we don't have to do `browser.new_context`, right?

Reply:
Not quite - it's more impactful than that.

The `new_context` fixture provided by `pytest-playwright` also makes sure to inject a bunch of default arguments to the `browser.new_context` call that were previously missing when I created it directly. The fixture also properly closes down any created contexts automatically; when creating them manually I should have been calling `context.close()` after the `yield` (this is why browser tabs were staying open during test runs in `--headed` mode before).

Reply:
Oooh.. neat!